src/app/functions/server/dashboard.ts

```diff
@@ -274,16 +357,52 @@ export function getDashboardSummary(
   // count unique breaches
   summary.dataBreachTotalNum = subscriberBreaches.length;
   const isBreachesOnly = summary.dataBrokerTotalNum === 0;
-  summary.sanitizedExposures = sanitizeExposures(
-    summary.allExposures,
-    summary.totalExposures,
+
+  // calculate total exposures
+  // summary.totalExposures = summary.dataBreachTotalExposuresNum + summary.dataBrokerTotalExposuresNum;
```

Comment:
Q: is this meant to be commented out?

Reply:
Thanks Peter, meant to delete.. deleted.